app/Providers/RouteServiceProvider.php

```diff
@@ -207,6 +207,20 @@ private function mapAdminRoutes()
 
             Route::resource('fares', 'FareController')->middleware('ability:admin,finances');
 
+            // downloads
+            Route::group([
+                'as'         => 'downloads.',
+                'prefix'     => 'downloads',
+                'middleware' => ['ability:admin, files'],
+            ], function () {
+                Route::get('/', 'DownloadController@index')->name('index');
+                Route::get('/create', 'DownloadController@create')->name('create');
+                Route::get('/edit/{id}', 'DownloadController@edit')->name('edit');
```

Comment:
Follow the Laravel conventions on these. I'm trying to wrap my head around how this is setup.

Reply:
Umm yeah, using it. Let me re-check the conventions.